src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalUtility.cs

```diff
@@ -98,7 +99,7 @@ public static void SaveBuildMessage(string fileName, BuildMessage bm)
             {
                 return;
             }
-            using (var writer = new StreamWriter(fileName))
+            using (var writer = new StreamWriter(fileName, false, Encoding.Unicode))
```

Comment:
>Unicode [](start = 75, length = 7)

tests failed for [http://docfx-ci-0.cloudapp.net/viewLog.html?buildId=12423&buildTypeId=DocfxCiWithScripts_DocfxCiForDevBranch](http://docfx-ci-0.cloudapp.net/viewLog.html?buildId=12423&buildTypeId=DocfxCiWithScripts_DocfxCiForDevBranch)